build-tools/scripts/omnisharp.json.in

```diff
@@ -0,0 +1,20 @@
+{
+    "msbuild": {
+        "loadProjectsOnDemand": true,
+    },
+
+    "RenameOptions": {
+        "RenameInComments": false,
+        "RenameOverloads": true,
+        "RenameInStrings": false
+    },
+
+    "sdk": {
+        "IncludePreleases": true,
+        "Path": "@DOTNET_SDK_PATH@",
```

Comment:
json does not like trailing commas at the end of the list. Same for Line 3.

Reply:
ah, damn, I keep forgetting that. Will fixicate